tutorials/shading/shading_reference/shading_language.rst

```diff
@@ -421,6 +421,24 @@ Uniforms can also be assigned default values:
     uniform vec4 some_vector = vec4(0.0);
     uniform vec4 some_color : hint_color = vec4(1.0);
 
+Global constants
+----------------
+
+Like the uniforms global constants are shared between all shader stages. However they are invisible outside of shader and must be initialized at declaration.
+The **const** keyword is used to make the variable to be a constant. All basic types, except samplers can be declared with that modifier.
```

Comment:
"Use the **const** keyword to declare a variable as a constant. All basic types, except samplers can be declared as constants. Constants are useful when you want to have access to a value throughout your shader that does not need to be modified. Accessing and using a constant value is slightly faster than using a uniform."